.github/workflows/localgov-drupal-ci.yml

```diff
@@ -43,3 +43,23 @@ jobs:
       - name: Run tests
         run: ./run-tests.sh
         shell: bash
+
+  MochaThemeTests:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x]
```

Comment:
+1.  I have never tried this "matrix" feature :)

Reply:
Yeah it's quite nice, should allow us to just add other node version we want to test in that list